gradle-build/action.yaml

```diff
@@ -35,9 +35,9 @@ runs:
       run: |
         if [ -z "${{ inputs.init_file }}" ]
         then
-          ./gradlew build --continue --no-daemon
-        else 
-          ./gradlew --init-script init.gradle.kts build --continue --no-daemon
+          ./gradlew clean build --continue --no-daemon
+        else
+          ./gradlew --init-script init.gradle.kts clean build --continue --no-daemon
```

Comment:
Actually can you fix my stupid mistake and change `init.gradle.kts` here to `${{ inputs.init_file }}`

Reply:
done